### PR TITLE
fix error handling on install_package

### PR DESCRIPTION
**Pull request by johnmccrae**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).
Signed-off-by: rishichawda <rishichawda@users.noreply.github.com>

### CODE REVIEW

**Review by PrajaktaPurohit — APPROVED** [review]
Looks good once we have had a chance to address the linter suggestions!

File: lib/chef/provider/package/powershell.rb [hunk 1]
```diff
@@ -55,8 +55,8 @@ def candidate_version
         def install_package(names, versions)
           names.each_with_index do |name, index|
             cmd = powershell_exec(build_powershell_package_command("Install-Package '#{name}'", versions[index]), timeout: new_resource.timeout)
-            next if cmd.nil?
-            raise Chef::Exceptions::PowershellCmdletException, "Failed to install package due to catalog signing error, use skip_publisher_check to force install" if /SkipPublisherCheck/.match?(cmd.error!)
+            next if !cmd.errors?
```
Comment (jaymzh) [inline lib/chef/provider/package/powershell.rb:58]:
```suggestion
            next unless cmd.errors?
```

File: omnibus/Gemfile.lock [hunk 2]
```diff
@@ -132,7 +132,7 @@ GEM
       train-winrm (>= 0.2.5)
       uuidtools (>= 2.1.5, < 3.0)
       vault (~> 0.16)
-      win32-api (~> 1.5.3)
+      win32-api (~> 1.5)
```
Comment (jaymzh) [inline omnibus/Gemfile.lock:135]: is this related to the bug? Should this be a separate PR?

**Review by jaymzh — CHANGES_REQUESTED** [review]
Small changes, please?

File: lib/chef/provider/package/powershell.rb (hunk 1 above)
Comment (johnmccrae) [reply lib/chef/provider/package/powershell.rb:58]: done

File: omnibus/Gemfile.lock (hunk 2 above)
Comment (johnmccrae) [reply omnibus/Gemfile.lock:135]: restoring that

**Comment by johnmccrae** [comment]
Already solved and sorted out. I was chasing my tail
